src/App.tsx

```diff
@@ -1,26 +1,47 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Component } from "react";
+import styles from "./App.module.scss";
+import Counter from "./components/Counter";
+import GenreSelect from "./components/GenreSelect";
+import SearchForm from "./components/SearchForm";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+interface AppState {
+	selectedGenre: string | null;
+}
+
+class App extends Component<{}, AppState> {
```

Comment:
Better go with functional components. Although React team state that class components woun't be removed in any future version of react, but still advice to use functional components in all new projects.